task/01-strings-tasks.js

```diff
@@ -221,9 +226,20 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+    let arr=[];
+    for (let count=0; count<str.length;count++){
+        if ((str.charCodeAt(count) >=78 & str.charCodeAt(count) <=90)| (str.charCodeAt(count) >=110 & str.charCodeAt(count)<=122))
```

Comment:
do not use "magic numbers' https://en.wikipedia.org/wiki/Magic_number_(programming)